### ISSUE TITLE
批曰。編譯有誤

### ISSUE THREAD

**Issue by cuixiping** [LingDong-/wenyan-lang#403]
## Describe your issue/suggestions

```
批曰。「「吾嘗觀「「易經」」之書」」
```
編譯得
```
/*"易經"*/
```

而應得
```
/* 吾嘗觀「「易經」」之書 */
```

另外，建議增加塊注釋(block comment)
```
長批曰。
    ……
    ……
    ……
長批至此。
```

## Checklist

Before you create this issue, Please make sure ALL the following items are checked

- [x] If this is a feature request, it's NOT listed [Here](https://github.com/LingDong-/wenyan-lang#feature-requests)
- [x] If this is a bug report, it's NOT listed [Here](https://github.com/LingDong-/wenyan-lang#known-bugs)

**Comment by LingDong-** [comment]
Hi @cuixiping , 

Nested 「「 are not supported yet, but will be among my very next things to add.

I think once we support that, then we already have block comments:

```
注曰「「
一行
一行
又一行
三四五六七八行
」」
```

Thanks!

**Comment by LingDong-** [comment]
Now nested comment is supported: 32b0f3abd1beb55cd28c369ad74b79e677248cc7
You can nest comments as much as you like:

```
批曰。「「「吾」嘗觀「「「易」經」」之「「「書」」」」」
```

However, the 「」's need to be matched even inside the comment. Otherwise there is no way for the parser to see where the comment ends.

Thanks!

**Comment by cuixiping** [comment]
Great!

**Comment by antfu** [comment]
Fixed in [v0.1.2](https://github.com/LingDong-/wenyan-lang/pull/416). Closing now. Feel free to reopen it if you got any questions.
